### DISCUSSION TITLE
Parenthesized list

### DISCUSSION THREAD

**Discussion by kyku** [jeremyevans/sequel#1947] [General]
Hi, I wonder if one can get something like "WHERE (foo, bar) IN ((1,2),(2,3),(3,4))" or "WHERE (a,b) > (b, c)" using Sequel DSL.

**Comment by kyku** [comment]
Ok, found out that Sequel::SQL::Builders#value_list is my friend here.

**Reply by jeremyevans** [reply]
`DB[:table].where([:foo, :bar]=>[[1, 2], [2, 3], [3, 4]])` and `DB[:table].where(Sequel[[:a, :b]] > [:b, :c])`
